collectd/map.jinja

```diff
@@ -1,15 +1,97 @@
-{% set collectd = salt['grains.filter_by']({
+{% set os_map = salt['grains.filter_by']({
     'Debian': {
         'pkg': 'collectd-core',
         'service': 'collectd',
         'config': '/etc/collectd/collectd.conf',
         'plugindirconfig': '/etc/collectd/plugins',
-        'javalib': '/usr/lib/collectd/java.so',
+        'TypesDB': ['/usr/share/collectd/types.db'],
     },
     'RedHat': {
         'pkg': 'collectd',
         'service': 'collectd',
         'config': '/etc/collectd.conf',
         'plugindirconfig': '/etc/collectd.d',
+        'TypesDB': ['/usr/share/collectd/types.db'],
```

Comment:
It was there before - I won't change it given that even things that **are** broken are preferred to be kept (cf. java.sls).

Additional changes might happen, but not here nor now.